src/release/upgrade.md

```diff
@@ -19,22 +19,67 @@ $ flutter upgrade
 This command gets the most recent version of the Flutter SDK
 that's available on your current Flutter channel.
 
-If you want an even more recent version of the Flutter SDK,
-switch to a less stable Flutter channel
+If you are using the **stable** channel
+and want an even more recent version of the Flutter SDK,
+switch to the **beta** channel using `flutter channel beta`,
 and then run `flutter upgrade`.
 
+### Keeping informed
+
+We publish [migration guides] for known breaking changes.
+
+We send announcements regarding these changes to the
+[Flutter announcements mailing list][flutter-announce].
+
+To avoid being broken by future versions of Flutter,
+consider submitting your tests to our [test registry].
+
 
 ## Switching Flutter channels
 
-Flutter has three [release channels][]:
-**stable**, **beta** and **master**.
+Flutter has two release channels:
+**stable** and **beta**.
 
-{{site.alert.info}}
-  The `dev` channel was retired as of Flutter 2.8.
-{{site.alert.end}}
+### The **stable** channel
+
+We recommend the **stable** channel
+for new users
+and for production app releases.
+It is updated roughly every three months,
+and may additionally receive occasional hot fixes
+for high-severity or high-impact issues.
+
+The Flutter team's plugins and packages are continually tested against the latest **stable** release.
```

Comment:
**issue:** Write in active voice per [GSG](https://developers.google.com/style/voice).
```suggestion
The Flutter team tests its plugins and packages against the latest **stable** release.
```

Reply:
We're not testing them, the computers test them. The idea is that the CI is using this branch to test the plugins. But I don't want to imply it's the only branch being used.

Reply:
gotcha. Changed to "The continuous integration for the Flutter team's plugins and packages includes testing against the latest stable release."